gsoc/background.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title:  "Background on GSoC: start here!"
+show_main: false 
+---
+
+# Background: Start Here!
+
+### What is GSoC? 
+
+Google Summer of Code is a remote summer coding program funded by 
+Google. Google distributes funds to open software development organizations to 
+recruit new developers and mentors for summer mentorships. You get paid to write 
+software at home (or wherever you like), with guidance from expert mentors 
+distributed around the globe.
+
+To apply to be a GSoC student, you must write an application to [GSoC] for one or
+more OpenAstronomy projects. Each summer, [several projects][oa projects] are offered by
+[OpenAstronomy mentors][oa mentors], with well-defined scope and pre-requisite
+experience. If you are selected by OpenAstronomy, you will be a GSoC student.
+
+### What is OpenAstronomy?
+
+OpenAstronomy is an umbrella organization which acts as an interface between
+the Google Summer of Code and several open source, open development projects
+in the astronomical community, including [astropy], [sunpy], and
+[several other projects in the astronomical community][oa members].
+It coordinates mentors and projects for Google Summer of Code mentorships 
+related to astronomy.
+
+### What is the application process?
+
+You must submit your application to the [Google Summer of Code][GSoC], but
+it's preferable that you share it before hand with your potential project mentor
+or the rest of the community. They will give you feedback before the deadline.
+Check the [application template][application] or look up for previous years
+applications on the projects' wikis.
```

Comment:
project has double meaning all along this page, would be nice to come up with more distinction between projects as gsoc projects and projects as OA members